conf/locale/locale_de-DE.ini

```diff
@@ -815,6 +815,7 @@ team_permission_desc=Welche Berechtigungsstufe soll das Team haben?
 
 form.name_reserved=Organisationsname '%s' ist bereits vergeben.
 form.name_pattern_not_allowed=Organisationsnamen der Form '%s' sind nicht erlaubt.
+form.create_org_not_allowed=Dieser Benutzer darf keine Organisation erstellen.
```

Comment:
I didn't find the translation service for gitea. There is only Crowdin for (gogs)[https://crowdin.com/project/gogs] and an empty project for (gitea)[https://crowdin.com/project/gitea].